modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/authenticator/management/v1/AuthenticatorSuccessTest.java

```diff
@@ -127,6 +131,38 @@ public void getAuthenticators() throws JSONException {
     }
 
     @Test(dependsOnMethods = {"getAuthenticators"})
+    public void testGetMetaTags() throws JsonProcessingException {
+
+        Response responseBefore = getResponseOfGet(AUTHENTICATOR_META_TAGS_PATH);
+        responseBefore.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("$", not(hasItem(CUSTOM_TAG)));
```

Comment:
Lets remove this as we dont need to validate "Custom" specifically and get it validated from the list

Reply:
The system have only deafault tags and check whether they are exists. This is to check whether cutsom is not exist before a custom authenticator created.